tests/e2e/Services/VCS/VCSBase.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Tests\E2E\Services\VCS;
+
+trait VCSBase
+{
```

Comment:
The VCS endpoints are supposed to be accessed only by the console right? 

Lets have two more classes VCSCustomServerTest.php VCSCustomClientTest.php

In the first, you try to access the vcs endpoints using an API key and it should fail with the appropriate error code. 
Similarly access with a regular user cookie in the latter.

Just 1 test each to make sure the route is not accessible.